tests/integration/test_dbapi_integration.py

```diff
@@ -153,7 +151,9 @@ def test_execute_many_without_params(trino_connection):
     cur = trino_connection.cursor()
     cur.execute("CREATE TABLE memory.default.test_execute_many_without_param (value varchar)")
     cur.fetchall()
-    cur.executemany("INSERT INTO memory.default.test_execute_many_without_param (value) VALUES (?)", [])
+    cur.executemany(
+        "INSERT INTO memory.default.test_execute_many_without_param (value) VALUES (?)", []
```

Comment:
Split query across lines instead of moving argument to newline

Reply:
Increasing the line length fixed those occurences.